source/PlayerInfoPanel.h

```diff
@@ -89,7 +94,7 @@ class PlayerInfoPanel : public Panel {
 	InfoPanelState panelState;
 
 	// Column headers that sort ships when clicked.
-	std::vector<ClickZone<InfoPanelState::ShipComparator *>> menuZones;
+	std::vector<ClickZone<PlayerInfoPanel::SortableColumn>> menuZones;
```

Comment:
Storing a copy of the SortableColumn object here doesn't seem like a good idea.
If you need access to the entire object, store a (const) pointer to it.

Reply:
Ahh good point.